frontend/src/components/common/form/Check.tsx

```diff
@@ -98,7 +98,10 @@ export const Check: React.FC<CheckProps> = ({
             value={!!checked}
             placeholder={placeholder}
             checked={!!checked}
-            onChange={() => setFieldValue(field, !checked)}
+            onChange={() => {
+              setFieldTouched(field);
```

Comment:
while debugging noticed that touching the check fields wasn't being recorded in formik.